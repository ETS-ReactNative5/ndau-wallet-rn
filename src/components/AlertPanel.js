import React, { Component } from 'react';

import { StyleSheet, Text, View } from 'react-native';

class AlertPanel extends Component {
  render() {
    return (
      <View style={styles.alertOuterContainer}>
        <View style={styles.alertContainer}>
          <Text style={styles.alertText}>{this.props.alertText}</Text>
        </View>
      </View>
    );
  }
}

var styles = StyleSheet.create({
  alertText: {
    color: '#4e957a',
    fontSize: 20,
    fontFamily: 'TitilliumWeb-Regular',
    padding: 10
  },
  alertContainer: {
    backgroundColor: '#c7f3e2',
    borderWidth: 2,
    borderColor: '#4e957a',
    borderRadius: 3
  },
  alertOuterContainer: {
    backgroundColor: '#c7f3e2',
    borderWidth: 1,
    borderColor: '#c7f3e2',
    borderRadius: 3,
    marginLeft: 20,
    marginRight: 20
  }
});

export default AlertPanel;
