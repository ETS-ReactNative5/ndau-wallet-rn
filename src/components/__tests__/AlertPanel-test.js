import { mount } from 'enzyme';
import React from 'react';
import { StyleSheet } from 'react-native';
import AlertPanel from '../AlertPanel';

import renderer from 'react-test-renderer';

describe('testing AlertPanel...', () => {
  let styles = StyleSheet.create({
    wizardText: {
      color: '#ffffff',
      fontSize: 20
    }
  });
  const navigator = {
    setStyle: () => {}
  };

  beforeEach(() => {});
  it('renders correctly', () => {
    const tree = renderer
      .create(<AlertPanel parentStyles={styles} navigator={navigator} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
