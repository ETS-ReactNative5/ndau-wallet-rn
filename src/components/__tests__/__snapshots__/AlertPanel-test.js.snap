// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing AlertPanel... renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#c7f3e2",
      "borderColor": "#c7f3e2",
      "borderRadius": 3,
      "borderWidth": 1,
      "marginLeft": 20,
      "marginRight": 20,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#c7f3e2",
        "borderColor": "#4e957a",
        "borderRadius": 3,
        "borderWidth": 2,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#4e957a",
          "fontFamily": "TitilliumWeb-Regular",
          "fontSize": 20,
          "padding": 10,
        }
      }
    />
  </View>
</View>
`;
